python/openvino_tokenizers/tokenizer_pipeline.py

```diff
@@ -839,18 +839,32 @@ def get_ov_subgraph(self, input_nodes):
 
         op_inputs = []
         input_nodes_iter = iter(input_nodes)
-        for node in self.inputs:
-            if isinstance(node, Sequence):
-                op_inputs.extend(islice(input_nodes_iter, 3))
-            elif isinstance(node, AddToken):
-                # Put a scalar as a ragged tensor with scalar shape and a single element
+        from itertools import groupby
```

Comment:
This import also has no reason to be local.

Reply:
moved to the top with other imports